src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -197,6 +190,10 @@ export default {
         }
     },
     computed: {
+        embedPreviewModalWidth() {
+            // User the iframe with as maximal width for the embed preview modal window
```

Comment:
there might be a couple misspelling in this comment, wouldn't it be more something like 
`Uses the iframe's width as maximal width for the entire modal window`?

Reply:
Done